Smart Search - move Filters (3x) from left to [Search Tools]

pe7er opened the pull request:
This PR moves the Filter option of all three views of **Smart Search** to the [Search Tools] button.
(See Issue #6941 regarding the inconsistency with Filter options).
## Test instructions

You have to enable the **Content - Smart Search** Plugin, and **Index your test website**
Please check the 3 different views before & after the patch: 
test all the filters and search box, clear results, choose max amount of results on a page.
## Smart Search: Manage Indexed Content
### Check the current position of the Filters

In back-end > Components > Smart Search > Indexed Content
The Filters are on the left hand side

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/9b6cf2caa2a5ca839c81409974b03437.png)
### This PR changes moves the Filter

from the left to the middle column and will be visible when you click on the [Search Tools] button.

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/741fb1b5b331b553bdbfc926edef7195.png)
## Smart Search: Manage Content Maps
### Check the current position of the Filters

In back-end > Components > Smart Search > Content Maps
The Filters are on the left hand side

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/7f31ad619fce230e4e1aaf825d1847f0.png)
### This PR changes moves the Filter

from the left to the middle column and will be visible when you click on the [Search Tools] button.

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/f2ed1d2e47e52ba1af0f66195a957e5f.png)
## Smart Search: Manage Search Filters
### Check the current position of the Filters

In back-end > Components > Smart Search > Search Filters
The Filters are on the left hand side

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/a2f3a221a31b8072071682887eabcefb.png)
### This PR changes moves the Filter

from the left to the middle column and will be visible when you click on the [Search Tools] button.

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/5314254c1b4b1a82f10485c12d4bff8d.png)
## Note

To Clear the Filters / Search box, you can use the [Clear] button.
However, sometimes you have to press 2 times to clear the results.
I don't know the cause and how to solve it...

pe7er committed:
Smart Search - move Filters (3x) from left to [Search Tools]

pe7er committed:
Added a space before "=" in 3 view.html.php files

jwaisner commented:
@test 

Patch works and filters work, however some behaviors I am not sure about.

Indexed Content->Search Tools->(Choose any filter)  Results in the filter content being collapsed back into "Search Tools" and you must click again to change filters.

Content Maps->Search Tools->(All, Published, Unpublished)  Results in the filter remaining displayed after content is filtered, however selecting "-Select Filter-" closes the filter options

Search Filters->Search Tools->(All, Published, Unpublished)  Results in the filter remaining displayed after content is filtered, however selecting "-Select Filter-" closes the filter options.

Question is: Which behavior is correct?
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6959">issues.joomla.org/joomla-cms/6959</a>.</sub>

brianteeman commented on administrator/language/en-GB/en-GB.com_finder.ini:
```diff
@@ -167,6 +167,8 @@ COM_FINDER_MAPS_CONFIRM_DELETE_PROMPT="Are you sure you want to delete the selec
 COM_FINDER_MAPS_MULTILANG="Note: Language filter system plugin has been enabled, so this branch will not be used."
 COM_FINDER_MAPS_NO_CONTENT="No results to display. Either no content has been indexed or no content meets your filter criteria."
 COM_FINDER_MAPS_RETURN_TO_BRANCHES="Return to Branches"
+COM_FINDER_MAPS_SELECT_BRANCHE="- Select Branche -"
```
This should be Branch not Branche

brianteeman commented:
All good to me except for a small typo in the new string which should be Branch not Branche<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6959">issues.joomla.org/joomla-cms/6959</a>.</sub>

pe7er commented:
Thanks for the feedback! I've to dig into it after this weekend...

jonnyefftek commented:
same issues:
- typo: "Branche"
- filters are collapsed after being applied<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6959">issues.joomla.org/joomla-cms/6959</a>.</sub>

m-b-o commented:
works fine, but the filters are collapsed after beeing applied
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6959">issues.joomla.org/joomla-cms/6959</a>.</sub>

pe7er commented:
Thank you for testing & all comments!
I corrected my alternative spelling of "Branch" ;-)
and corrected the issue with the Search Tools filter: now it should remain opened after selecting a filter.

pe7er committed:
Keep Search Filters open after selection

jduerscheid commented:
Test was successful. No collapsed after beeing applied. #jab15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6959">issues.joomla.org/joomla-cms/6959</a>.</sub>

zero-24 commented:
This just needs on more test as we have now fixed this issues :smile: @ChrisBreaks @m-b-o @jonnyefftek <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6959">issues.joomla.org/joomla-cms/6959</a>.</sub>

jwaisner commented:
@test 

Tested successfully. Previous behavior of collapsing has been resolved.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6959">issues.joomla.org/joomla-cms/6959</a>.</sub>

zero-24 commented:
RTC based on testing :smile: Thanks :+1: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6959">issues.joomla.org/joomla-cms/6959</a>.</sub>

roland-d commented:
@pe7er Can you please fix the merge conflicts?

pe7er commented:
I've created a new PR #8140 to fix the merge conflict

roland-d commented:
Closing this PR as we have a new one #8140
